src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -111,8 +111,18 @@
 Application Source Code was disclosed by the web server
 
 <H2>Strict Transport Security Header Scanner</H2>
-This scanner checks HTTPS response headers for the presence of a HTTP Strict Transport Security header (alerts if it is not found).
-Also checks if the HTTP Strict Transport Security (HSTS) header contains a max-age directive with value 0, which instructs clients to reset the entire HSTS Policy associated with the HSTS Host.
+This scanner checks HTTPS response headers for the presence of a HTTP Strict Transport Security (HSTS) header and tests for various implementation concerns, 
+alerting if they're found.
+
+Alerts generated:
```

Comment:
The entries do not match the names of the alerts, is that on purpose? (it's fine to me just want to be sure).